### DISCUSSION TITLE
System-wide text-based profile defaults configs?

### DISCUSSION THREAD

**Discussion by tsilvs** [zen-browser/desktop#1164] [Ideas]
I need something as powerful as `/etc/firefox/policies/policies.json` to set default search engines, keywords & pre-installed addon selection. But most of policies are available on Firefox ESR only and will probably never be ported to Librewolf.

**Comment by Mecallie** [comment]
If you want to do this via a GPO: use the attached .zip.
I don't know about the .json option, but in the admx templates it was as easy as replacing "Firefox" with "Zen" (well, almost). The browser supports all the policies Firefox does, you just need to get the naming correct.
